pkg/apis/feeds/v1alpha1/cluster_event_source_validation.go

```diff
@@ -25,19 +25,6 @@ func (ces *ClusterEventSource) Validate() *apis.FieldError {
 }
 
 func (cess *ClusterEventSourceSpec) Validate() *apis.FieldError {
+	// nothing to validate
```

Comment:
nit: I find this comment odd, since we then do call validate on the CESS.

Reply:
I mean noting to validate for fields that are managed directly by ClusterEventSourceSpec. I suppose I can just remove the comment and continue `cess.CommonEventSourceSpec.Validate()`